### PR TITLE
registry: Fill in all the docker commands into the various pages

### PR DESCRIPTION
**Pull request by stefwalter**
Currently we use the registry host listed in the ImageStream object, which may be incorrect externally. Later we'll have a place to either retrieve or recieve this configuration.

### CODE REVIEW

**Comment by stefwalter** [comment]
Waiting on #3451 and #3456

**Comment by stefwalter** [comment]
Fixed to use '-p' instead of '-t' in 'docker login' command.

File: pkg/registry/scripts/main.js [hunk 1]
```diff
@@ -93,11 +99,21 @@
...
                 $scope.curtains = { };
-                discover().then(function() {
+                discover().then(function(options) {
                     $scope.curtains = null;
+
+                    /* See if we have a bearer token to use */
+                    var authorization;
+                    if (options.headers) {
+                        authorization = options.headers['Authorization'] || "";
+                        $scope.config.registry.password = authorization.split("earer ")[1];
```
Comment (petervo) [inline pkg/registry/scripts/main.js]: Makes no difference, but "Bearer" is probably more readable.

**Comment by petervo** [comment]
Does the new dashboard code also need the imagestream/stream changes?

File: pkg/registry/scripts/main.js (hunk 1 above)
Comment (stefwalter) [inline pkg/registry/scripts/main.js]: True, I was worried about capitalization. But I guess I should just handle that correctly.

**Comment by stefwalter** [comment]
> Does the new dashboard code also need the imagestream/stream changes?

Good catch, I'll do that to be consistent.

File: pkg/registry/scripts/main.js (hunk 1 above)
Comment (stefwalter) [inline pkg/registry/scripts/main.js]: Done.

**Comment by stefwalter** [comment]
Ready for review again.

**Pull request closed without merging**